pkg/integration/ttl_integration_test.go

```diff
@@ -31,14 +31,11 @@ func (log logger) Warnf(format string, args ...interface{})  { log.t.Logf(format
 func (log logger) Errorf(format string, args ...interface{}) { log.t.Logf(format, args...) }
 
 func TestTTLResponse(t *testing.T) {
-
```

Comment:
Ran this file through a formatter hence these changes.

Reply:
Can we make a separate PR with this, and just land it?